source_code/main_mcu/src/NODEMGMT/nodemgmt.h

```diff
@@ -220,7 +220,8 @@ typedef struct
     uint8_t credential_id[16];      // Credential identifier. A probabilistically-unique byte sequence identifying a public key credential source (tag in solokey). Generated by device
     uint8_t user_handle[64];        // An opaque byte sequence with a maximum size of 64 bytes. User handles are not meant to be displayed to users. Generated by RP
     uint8_t user_handle_len;        // Actual length of user handle
-    uint8_t reserved[2];            // Reserved
+    uint8_t keyType;                // Key Type for FIDO2 (Currently supported is ES256=1 and EDDSA=2)
```

Comment:
we have quite an issue here... units are already in the field with created credentials!
It seems that by default we're setting that field to 0: https://github.com/mooltipass/minible/blob/master/source_code/main_mcu/src/LOGIC/logic_database.c#L898 so you could just define ES256 to 0 ? :)